docs/source/api/python/mantid/plots/index.rst

```diff
@@ -55,16 +85,16 @@ To generate a 1D plots of some spectra.
    :figwidth: image
 
 This example demonstrates adding multiple spectra onto a single 1D
-plot and overriding some of the default behavior. :func:`~mantid.plots.plot` is a normal
-line plot, while :func:`~mantid.plots.errorbar` adds the uncertainties. It should be
+plot and overriding some of the default behavior. :func:`~mantid.plots.MantidAxes.plot` is a normal
+line plot, while :func:`~mantid.plots.MantidAxes.errorbar` adds the uncertainties. It should be
 warned that every call to one of the plot functions will automatically
 annotate the axes with the last one called being the one that takes
 effect.
 
-Two common ways to look at 2D plots are :func:`~mantid.plots.contourf` and
-:func:`~mantid.plots.pcolormesh`. The difference between these is the
-:func:`~mantid.plots.contourf` calculates smooth lines of constant
-value, where the :func:`~mantid.plots.pcolormesh` is the actual data
+Two common ways to look at 2D plots are :func:`~mantid.plots.MantidAxes.contourf` and
```

Comment:
Do you want to add a snippet from the matplotlib docs: "pcolormesh is similar to pcolor(), but uses a different mechanism and returns a different object; pcolor returns a PolyCollection but pcolormesh returns a QuadMesh. It is much faster, so it is almost always preferred for large arrays."